packages/component-library/src/ButtonNew/ButtonNew.test.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
```

Comment:
You can remove this test file for now, these failing tests are causing Travis to fail